packages/client/src/clients/feed/feed.ts

```diff
@@ -6,333 +6,333 @@ import Knock from "../../knock";
 import { NetworkStatus, isRequestInFlight } from "../../networkStatus";
 
 import {
-  FeedClientOptions,
-  FeedItem,
-  FeedMetadata,
-  FeedResponse,
-  FetchFeedOptions,
+	FeedClientOptions,
```

Comment:
Jeff is it possible to tweak your editor setting so we can keep things formatted the same / consistently? That will help clean up the diff, but also I think we want to keep all of our public ts/js formatted consistently.

Reply:
Yeah, I'm happy to tweak whatever is needed, but I've already go Prettier installed and thought this should pick up the prettier-config file in this repo. If you could also sanity check mere here, but if I look at these two files I can't see any of the formatting differences expressed in the diff. They both look the same to me: https://github.com/knocklabs/javascript/blob/fix/je-update-status-event/packages/client/src/clients/feed/feed.ts
https://github.com/knocklabs/javascript/blob/main/packages/client/src/clients/feed/feed.ts